src/main/java/com/google/sps/data/AvailabilityTimeSlot.java

```diff
@@ -57,6 +60,8 @@ static Builder builder() {
 
     abstract Builder setSelected(boolean selected);
 
+    abstract Builder setScheduled(boolean scheduled);
```

Comment:
What's the difference between selected vs scheduled?
Update: l know what they are but only after looking at some comment 7 files down. 
Could you put a comment here differentiating between the two?

Reply:
Done